### DISCUSSION TITLE
Config help: Renovate stops raising PRs

### DISCUSSION THREAD

**Discussion by JrobT** [renovatebot/renovate#15340] [Request Help] [answered]
redacted

**Comment by rarkins** [comment, marked as answer]
It looks to only be the "Pin Dependencies" PR which is blocked. Not "all updates". Do you see anything to make you think it's all updates being blocked?

**Comment by JrobT** [comment]


**Reply by rarkins** [reply]
I don't think the problem is what you think it is. It says 0 branches, and that is highly unlikely to be caused by any declined PR in the past

**Reply by JrobT** [reply]
Fair enough. Do you have an idea of what I should be looking for?

**Reply by rarkins** [reply]
There is a debug log message `packageFiles with updates` which logs all found package files, dependencies, and if any updates are pending. Are there updates shown there?

**Reply by rarkins** [reply]
None of them are truly updates - they're just pins.

Declining a Pin dependencies PR disables future ones forever, unless you rename the PR. In Bitbucket renaming declined PRs is not possible, so if you have declined it but then regret it, you'll need to pin manually.

In your original post, your expectations were correct:

> The specific issue I'm seeing is that if Renovate raises a PR which is then 'Declined' (& the branch deleted) I would expect this to mean Renovate no longer tries to push PRs for that changes unless the title of the PR changes. 

But this observation seems wrong:

> What seems to happen is that Renovate doesn't raise any more PRs until the 'Declined' PR is merged (for any repositories).

Because there's *only* pin dependencies showing in the log above

**Reply by JrobT** [reply]
Right - thank you, that makes sense. Thank you for your time in answering my questions, I appreciate it.
